src/views/Home.jsx

```diff
@@ -19,6 +49,23 @@ export function Home({ data, setListPath }) {
 						/>
 					))}
 			</ul>
+
+			<form onSubmit={handleSubmit}>
+				<h3>Create New Shopping List</h3>
+				<label htmlFor="newListName">Name Your List</label>
+				<br />
+				<input
+					type="text"
```

Comment:
![Screenshot 2024-08-23 at 4 22 35 PM](https://github.com/user-attachments/assets/bfa84280-08a7-4754-ba03-31579ac88673)

suggestion: I noticed an issue with the label tag in my console. I think I am just reading it as the prop `htmlFor` for the `label` tag references the input's `id` prop not the `name` prop so the `id` prop would just be added.

```suggestion
<input
    id="newListName"
```

Reply:
Ah! Good call out. Thanks for catching that.